src/app/actions/swap.js

```diff
@@ -97,6 +99,29 @@ export const createOrder = (swap) => (dispatch) => Promise.resolve().then(() =>
   })
 })
 
+export const createManualOrder = (swap) => (dispatch) => Promise.resolve().then(() => {
+  if (swap.error) return swap
+  const finish = dispatch(createSwapFinish('createOrder', swap))
+  const { receiveAddress, refundAddress, sendAmount, sendSymbol, receiveSymbol, id } = swap
+  const userId = receiveAddress
```

Comment:
We use the userId parameter for the `sendWalletId` so manual swaps should omit `userId` altogether until we add connected wallet support